### ISSUE TITLE
Slack Alerts Bug

### ISSUE THREAD

**Issue by Bosco-2019** [blockpane/tenderduty#66]
Hey, 

Started using Tenderduty to push alerts to Slack however, alerts seem to bug out on Slack only as we have alerts going to Discord also from the same instance and they're fine.

Issues:
Regardless of the value passed into node_down_alert_minutes: in the config slack will alert with any node downtime even if it is only 30 seconds Discord does not. 

Alerts are repetitive and do not clear where Discord will alert once as it should slack send 30+ messages for the same alert and then again 30+ when the alert is resolved.

**Comment by blockpane** [comment]
Yes, there are a bunch of slack bugs. I'm not sure when I'll get around to fixing it, but I will fix it since I primarily use slack for alerts too and am rather annoyed by some of the bugs.
